### ISSUE TITLE
Let me select my own time limit

### ISSUE THREAD

**Issue by newmanne** [sjpfenninger/ipython-extensions#3]
Hi, I'm really enjoying this extension. But I hate the preset time intervals - I would love to be able to set a timeout of e.g. 90 seconds. Either more options or else a way to enter my own option would be great.

**Comment by sjpfenninger** [comment]
What options would you want (I take it that 90 seconds is one)? Adding more would be straightforward, allowing to enter own option (in addition to dropdown) probably more work than I have time to put in these days

**Comment by newmanne** [comment]
Fair enough. I think 90 seconds and 3 minutes would be good to have. Past 3 minutes I've almost definitely switched to doing something else and will want the notification.
